nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -663,7 +663,7 @@ def to_csv(self):
     "relationships",
     "webhooks",
 )
-class ContactLCM(PrimaryModel):
+class ContactLCM(PrimaryModel):  # pylint: disable=too-many-ancestors
```

Comment:
I'd vote for disabling too-many-ancestors globally, as I don't think it's ever a useful thing to warn about.

Reply:
That'll make it much easier, I'll update Pylint config.